### ISSUE TITLE
Add Code of Conduct

### ISSUE THREAD

**Issue by M0nica** [Rebuild-Black-Business/RBB-Website#216]
Can a formal Code of Conduct be added to this project? https://help.github.com/en/github/building-a-strong-community/adding-a-code-of-conduct-to-your-project 

Also, potentially community guidelines as well: https://help.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors

**Comment by racedale** [comment]
We had this default one in the original volunteer docs repo.  Should be able to adjust it for this easily

https://github.com/Rebuild-Black-Business/RBB-Volunteer-Docs/blob/master/Code_Of_Conduct.md

**Comment by mbifulco** [comment]
Suggestion: make a `Rebuild-Black-Business/.github` repo, and move these files there, to take advantage of [GitHub's default community health settings](https://help.github.com/en/github/building-a-strong-community/creating-a-default-community-health-file)

**Comment by M0nica** [comment]
@mbifulco I think using the community health settings would be helpful to better surface the code of conduct and have it added to new repos by default. 

@racedale glad to know there's already one on GitHub within in the organization!

**Comment by M0nica** [comment]
although, it looks like it'll have to be setup per repo a bit but it's probably good to have it as a default as well
`If a repository has any files in its own .github/ISSUE_TEMPLATE folder, including issue templates or a config.yml file, none of the contents of the default .github/ISSUE_TEMPLATE folder will be used.`

**Comment by magnificode** [comment]
Hey @M0nica - I created a Community Health repo here that includes the Code of Conduct, and Contributing guidelines here: https://github.com/Rebuild-Black-Business/.github

Would love your input on that initial pass, and very much appreciate you making sure we've got the Code of Conduct properly documented here!

I've opened up #260 to make sure that the Code of Conduct markdown file is added in this project. Thanks again @M0nica!
